SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -2,22 +2,16 @@
 
 #include "LocalDeploymentManager.h"
 
-#include "AssetRegistryModule.h"
```

Comment:
Are these removed as they are already included via other includes or are these removed as we do not use any code from these headers?

Reply:
don't use any code from those headers

Reply:
Cool, if that is the case we can remove these. We usually try not to rely on headers being included from other headers and ensure we are explicit about every header we need.